### PR TITLE
Release 2.5.7.1

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://github.com/NVIDIA/nccl/archive/v{{ version }}-{{ revision }}.tar.gz
```
Comment (CJ-Wright) [inline recipe/meta.yaml]:
```suggestion
  url: https://github.com/NVIDIA/nccl/archive/v{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% set name = "nccl" %}
-{% set version = "2.5.6" %}
+{% set version = "2.5.7" %}
```
Comment (CJ-Wright) [inline recipe/meta.yaml]:
```suggestion
{% set version = "2.5.7-1" %}
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 {% set revision = "1" %}
```
Comment (CJ-Wright) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by jakirkham** [comment]
Let's move this into a new PR so we can test and see if the bot picks it up 😉

**Comment by jakirkham** [comment]
Please see PR ( https://github.com/conda-forge/nccl-feedstock/pull/17 ).

**Comment by jakirkham** [comment]
Superseded by PR ( https://github.com/conda-forge/nccl-feedstock/pull/18 ).

**Pull request closed without merging**